packages/ui/components/form/select/Select.tsx

```diff
@@ -45,7 +45,7 @@ export const getReactSelectProps = <
   menuPlacement?: MenuPlacement;
 }) => ({
   menuPlacement,
-  className: classNames("block h-[36px] w-full min-w-0 flex-1 rounded-md", className),
+  className: classNames("block h-[38px] w-full min-w-0 flex-1 rounded-md", className),
```

Comment:
Had to increase this height because the `ControlComponent` minimum height is `38px`

<img width="575" alt="Screenshot 2023-01-27 at 2 43 25 PM" src="https://user-images.githubusercontent.com/121884634/215047192-eb54dab2-aefa-4bad-bec4-af0d6abfc44d.png">

Reply:
@sean-brydon is this risky?

Reply:
Yeah we should ideally change the control component to have a min-height of 36. 

Not 100% sure why its 38 off the top of my head but im pretty sure design spec is 36px for height

Reply:
If it's supposed to be 36px then we can try fixing it

Reply:
Yeah i think this needs to be addressed before hand as all selects are now 38px in height 
<img width="397" alt="CleanShot 2023-02-01 at 10 31 11@2x" src="https://user-images.githubusercontent.com/55134778/216018763-02c19c25-e269-4688-b252-da541716e603.png">

Reply:
we will look into this